lib/proportions.js

```diff
@@ -64,9 +64,8 @@ define(['d3-interpolate', 'snabbdom', 'filters/genericnode', 'helper'],
           var th = V.h('th', a);
           var td = V.h('td', V.h('span', {
             style: {
-              width: Math.round(v * 100) + '%',
-              backgroundColor: scale(v),
-              color: 'white'
+              width: 'calc(25px + ' + Math.round(v * 90) + '%)',
```

Comment:
One question:
`25px != .5em` where you got `25px` ?
https://github.com/ffrgb/meshviewer/blob/future/scss/modules/_proportion.scss#L22

the `90` looks like found in tests to

Reply:
This is no padding. and max-width is around 250 px - 10% is 25px.Adding 25px to every bar by default the issue should resolve

Reply:
build it -> looks good